### PR TITLE
ENH - New ``skglm`` README

### LINKED ISSUE #83: DOC update citation in README and doc
we are no longer a `@online` but a `@inproceedings` :)

### PR DESCRIPTION
**Pull request by Badr-MOUFAD**
This proposes a new README of the package.
Main improvement:

- Reformulation of ``skglm`` presentation and features
- Add Get started and Contribute sections

It also updates the citation section and thereby closes #83 

---------------
A [live version](https://github.com/Badr-MOUFAD/skglm/tree/new-readme#readme) of the README

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 DISTNAME = 'skglm'
 DESCRIPTION = 'A fast and modular scikit-learn replacement for generalized linear models'
-with open('README.rst', 'r') as f:
+with open('README.md', 'r', encoding='utf-8') as f:
```
Comment (Badr-MOUFAD) [inline setup.py:16]: Not adding ``encoding`` will result in reading incorrectly the file. Especially here, I'm using emojis.

cf. https://github.com/mathurinm/celer/pull/277

File: README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+pip install -U skglm
+```
+
+It is also available on Conda _(not yet, but very soon...)_ and can be installed via the command
+
+```shell
+conda install skglm
+```
+
+## Trying ``skglm``
```
Comment (PABannier) [inline README.md:50]: I'd use "First steps with ``skglm``"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+estimator = MCPRegression()
+estimator.fit(X, y)
+
+# print R²
+print(estimator.score(X, y))
+```
+You can refer to the documentation to explore the list of ``skglm``'s already-made estimators. 
+
+Didn't find one that suits you :monocle_face:, you still can compose your own.
+Here is a code snippet that fits the previous estimator with a Huber loss instead.
```
Comment (PABannier) [inline README.md:72]: A bit confusing. I'd say `Here is a code snippet that fits a MCP-regularized problem with Huber loss`.

**Comment by Klopfe** [comment]
Very nice work @Badr-MOUFAD, it looks awesome. I fixed some typos and the rest looks good to me.

**Review by mathurinm — APPROVED** [review]

**Review by PABannier — APPROVED** [review]
Thanks @Badr-MOUFAD ! Really nice work :)

**Pull request merged**